apps/condo/pages/meter/index.tsx

```diff
@@ -126,41 +128,40 @@ const MetersPage: IMeterIndexPage = () => {
     usePreviousQueryParams({ paramNamesForPageChange: ['tab', 'type'], trackedParamNames: ['sort', 'filters', 'isShowActiveMeters', 'isShowArchivedMeters'], employeeSpecificKey: employeeId }) 
 
     const { tab } = parseQuery(router.query)
-    const type = get(router.query, 'type', METER_TYPES.unit) as string
+    const type = Array.isArray(get(router.query, 'type')) ? undefined : get(router.query, 'type') as string
 
     const tabAsMeterPageType = tab ? MeterPageTypeFromQuery(tab) : null
-    
-    const activeTab = useMemo(() => AVAILABLE_TABS.includes(tabAsMeterPageType) ? tabAsMeterPageType : get(AVAILABLE_TABS, [0], ''),  [tabAsMeterPageType])
-    const activeType = useMemo(() => type ? type : METER_TYPES.unit,  [type])
 
-    const changeRouteToActiveTab = useCallback(async (activeTab: string) => {
-        router.replace({ query: { tab: activeTab, type: get(router, 'query.type') } })
-    }, [router])
+    const activeTab = useMemo(() => AVAILABLE_TABS.includes(tabAsMeterPageType) ? tabAsMeterPageType : get(AVAILABLE_TABS, [0]),  [tabAsMeterPageType])
+    const activeType = useMemo(() => type in METER_TYPES ? type : METER_TYPES.unit, [type])
 
-    const changeRouteToActiveType = useCallback(async (activeType: string) => {
-        router.replace({ query: { tab: get(router, 'query.tab'), type: activeType } })
+    const changeRouteToActiveParams = useCallback(async (newParameters) => {
+        await updateQuery(router, { newParameters }, { resetOldParameters: true, routerAction: 'replace' } )
     }, [router])
 
 
     useEffect(() => {
-        if (!activeTab) return
-        if (!tab || tab !== activeTab) {
-            changeRouteToActiveTab(activeTab)
+        if (!activeType && !activeTab) return
+        if ((!type || type !== activeType) && (!tab || tab !== activeTab)) {
```

Comment:
```suggestion
        if ((!type || type !== activeType) || (!tab || tab !== activeTab)) {
```
It seems that if we now reset the parameters when changing the route, then we can leave only one such condition